tests/Integrations/Lumen/V5_6/TraceSearchConfigTest.php

```diff
@@ -30,14 +32,14 @@ public function testScenario()
             $this->call(GetSpec::create('Testing trace analytics config metric', '/simple'));
         });
 
-        $this->assertExpectedSpans(
+        $this->assertFlameGraph(
             $traces,
             [
                 SpanAssertion::build(
                     'lumen.request',
                     'lumen',
                     'web',
-                    'App\Http\Controllers\ExampleController@simple simple_route'
+                    'GET simple_route'
```

Comment:
Same as in [this comment](https://github.com/DataDog/dd-trace-php/pull/884/files#r426516942)